Fix wvstreams so that it builds with uClibc: we don't have execinfo.h,
so we can't do backtrace() stuff.

Signed-off-by: Simon Dawson <spdawson@gmail.com>

diff -Nurp a/utils/wvcrash.cc b/utils/wvcrash.cc
--- a/utils/wvcrash.cc	2008-12-17 12:24:20.000000000 +0000
+++ b/utils/wvcrash.cc	2012-07-27 22:00:15.456502262 +0100
@@ -28,7 +28,9 @@
 // FIXME: this file mostly only works in Linux
 #ifdef __linux
 
-# include <execinfo.h>
+#ifdef HAVE_EXECINFO_H
+#include <execinfo.h>
+#endif
 #include <unistd.h>
 
 #ifdef __USE_GNU
@@ -267,9 +269,11 @@ static void wvcrash_real(int sig, int fd
 	}
     }
 
+#ifdef HAVE_EXECINFO_H
     wr(fd, "\nBacktrace:\n");
     backtrace_symbols_fd(trace,
 		 backtrace(trace, sizeof(trace)/sizeof(trace[0])), fd);
+#endif
     
     if (pid > 0)
     {
